src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -8,6 +8,10 @@ import { formatFriendlyDateToNow } from "components/utils";
 import { generateFakeAuditLogs } from "pages/Logs/audit-logs-fake-data";
 import getUserName from "utils/getUserName";
 
+type Props = {
+  showModel: boolean;
```

Comment:
This could be an optional prop so that we don't have to provide it when we don't want to show the model.